docs/install/README.md

```diff
@@ -371,38 +373,451 @@ Status:
+K8ssandra Operator requires clusters to have routable pod IPs. kind clusters by default 
+will run on the same Docker network which means that they will have routable IPs.
+
+#### Create kind clusters
+Run `setup-kind-multicluster.sh` as follows:
+
+```sh
+./setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
+```
+
+When creating a cluster, kind generates a kubeconfig with the address of the api server 
+set to localhost. We need a kubeconfig that has the api server address set to its 
+internal ip address. `setup-kind-multi-cluster.sh` takes care of this for us. Generated 
+files are written into a `build` directory.
+
+Run `kubectx` without any arguments and verify that you see the following contexts 
+listed in the output:
+
+* kind-k8ssandra-0
+* kind-k8ssandra-1
+
+#### Install the control plane
+We will install the control plane in `kind-k8ssandra-0`. Make sure your active context 
+is configured correctly:
+
+```console
+kubectx kind-k8ssandra-0
+```
+
+Install the operator:
+
+```console
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator --create-namespace
+```
+
+This `helm install` command does the following:
+
+* Create the `k8ssandra-operator` namespace if necessary
+* Install Cass Operator in the `k8ssandra-operator` namespace
+* Install K8ssandra Operator in the `k8ssandra-operator` namespace
+
+This does not currently install Cert Manager. Cass Operator requires Cert Manager when
+its webhook is enabled. This installs with Cass Operator's webhook disabled.
+
+Verify that the Helm release is installed:
+
+```console
+helm ls -n k8ssandra-operator
+```
+
+```console
+NAME              	NAMESPACE         	REVISION	UPDATED                             	STATUS  	CHART                    	APP VERSION
+k8ssandra-operator	k8ssandra-operator	1       	2021-09-30 16:28:08.722822 -0400 EDT	deployed	k8ssandra-operator-0.31.0	1.0.0
+```
+
+Verify that the following CRDs are installed:
+
+* `cassandradatacenters.cassandra.datastax.com`
+* `clientconfigs.k8ssandra.io`
+* `k8ssandraclusters.k8ssandra.io`
+* `replicatedsecrets.k8ssandra.io`
+* `stargates.k8ssandra.io`
+
+Check that there are two Deployments. The output should look similar to this:
+
+```console
+kubectl -n k8ssandra-operator get deployment
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+k8ssandra-operator-cass-operator        1/1     1            1           85s
+k8ssandra-operator-k8ssandra-operator   1/1     1            1           85s
+```
+
+The operator looks for an environment variable named `K8SSANDRA_CONTROL_PLANE`. When set 
+to `true` the control plane is enabled. It is enabled by default.
+
+Verify that the `K8SSANDRA_CONTROL_PLANE` environment variable is set to `true`:
+
+```console
+kubectl get deployment k8ssandra-operator-k8ssandra-operator -o jsonpath='{.spec.template.
+spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
```

Comment:
```suggestion
kubectl get deployment k8ssandra-operator-k8ssandra-operator -n k8ssandra-operator -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
```

The namespace was missing, which made the command fail.

Reply:
Nice catch. Thanks.